README.md

```diff
@@ -212,29 +214,30 @@ The table below compares the performance (measured in terms of average elapsed t
 
 |  distance  |  loop  |  colwise  |  gain  |
 |----------- | -------| ----------| -------|
-| SqEuclidean | 0.005460s |  0.001676s |  3.2582 |
-| Euclidean | 0.005513s |  0.001681s |  3.2792 |
-| Cityblock | 0.005409s |  0.001675s |  3.2292 |
-| Chebyshev | 0.008592s |  0.004575s |  1.8779 |
-| Minkowski | 0.056741s |  0.048808s |  1.1625 |
-| Hamming | 0.005320s |  0.001670s |  3.1847 |
-| CosineDist | 0.005663s |  0.001697s |  3.3378 |
-| CorrDist | 0.010000s |  0.013904s |  0.7192 |
-| ChiSqDist | 0.009626s |  0.004734s |  2.0333 |
-| KLDivergence | 0.046696s |  0.035091s |  1.3307 |
-| RenyiDivergence | 0.021123s |  0.012006s |  1.7594 |
-| RenyiDivergence | 0.080503s |  0.066987s |  1.2018 |
-| JSDivergence | 0.066404s |  0.059564s |  1.1148 |
-| BhattacharyyaDist | 0.013065s |  0.008807s |  1.4836 |
-| HellingerDist | 0.013013s |  0.008679s |  1.4993 |
-| WeightedSqEuclidean | 0.005534s |  0.001676s |  3.3028 |
-| WeightedEuclidean | 0.005601s |  0.001723s |  3.2513 |
-| WeightedCityblock | 0.005496s |  0.001675s |  3.2815 |
-| WeightedMinkowski | 0.057847s |  0.051389s |  1.1257 |
-| WeightedHamming | 0.005439s |  0.001673s |  3.2513 |
-| SqMahalanobis | 0.134717s |  0.019530s |  6.8980 |
-| Mahalanobis | 0.129455s |  0.020114s |  6.4361 |
-| BrayCurtis | 0.005666s |  0.001680s |  3.3736 |
+| SqEuclidean | 0.005488s |  0.001032s |  5.3198 |
+| Euclidean | 0.005455s |  0.001033s |  5.2790 |
+| Cityblock | 0.005527s |  0.001047s |  5.2812 |
+| Chebyshev | 0.009622s |  0.005049s |  1.9058 |
+| Minkowski | 0.046888s |  0.040620s |  1.1543 |
+| Hamming | 0.005506s |  0.001057s |  5.2077 |
+| CosineDist | 0.007194s |  0.002282s |  3.1530 |
+| CorrDist | 0.012108s |  0.014342s |  0.8442 |
+| ChiSqDist | 0.006355s |  0.001259s |  5.0481 |
+| KLDivergence | 0.031378s |  0.024853s |  1.2625 |
+| RenyiDivergence | 0.017307s |  0.009944s |  1.7406 |
+| RenyiDivergence | 0.051003s |  0.045333s |  1.1251 |
+| JSDivergence | 0.046896s |  0.042365s |  1.1070 |
+| BhattacharyyaDist | 0.008461s |  0.003804s |  2.2244 |
+| HellingerDist | 0.008584s |  0.003678s |  2.3341 |
+| WeightedSqEuclidean | 0.005481s |  0.001182s |  4.6370 |
+| WeightedEuclidean | 0.005591s |  0.001198s |  4.6680 |
+| WeightedCityblock | 0.005712s |  0.001177s |  4.8521 |
+| WeightedMinkowski | 0.048870s |  0.041635s |  1.1738 |
+| WeightedHamming | 0.005682s |  0.001183s |  4.8042 |
+| PeriodicEuclidean | 0.012928s |  0.007914s |  1.6335 |
```

Comment:
Put it with `Euclidean` for consistency.

Reply:
Yes, I need to run the benchmarks again. This is still the from the early submission, when I tried to follow closely the weighted metrics code.